.evergreen/config.yml

```diff
@@ -97,7 +98,7 @@ functions:
             # If this was a patch build, doing a fresh clone would not actually test the patch
             cp -R ${PROJECT_DIRECTORY}/ $DRIVERS_TOOLS
           else
-            git clone https://github.com/mongodb-labs/drivers-evergreen-tools.git $DRIVERS_TOOLS
+            git clone --branch GODRIVER-2607 https://github.com/blink1073/drivers-evergreen-tools.git $DRIVERS_TOOLS
```

Comment:
did we want to use this specific branch?

Are we waiting for a merge with master before we can leverage the mongo-labs master branch?

Reply:
Ignore. I just saw in the PR comments!